### PR TITLE
Rename event creation classes.

### LINKED ISSUE #9166: Tipset Renaming
There are a large number of files with the prefix `Tipset`. This was done because when the tipset algorithm was developed, there were alternate event creation algorithms. Now that we have removed all other event creation algorithms, we can rename classes to remove the `Tipset` prefix from many places.

### PR DESCRIPTION
**Pull request by cody-littley**
Closes #9166

### CODE REVIEW

**Review by edward-swirldslabs — DISMISSED** [review]

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java [hunk 1]
```diff
@@ -771,13 +771,13 @@ private void initializeState(@NonNull final SignedState signedState) {
     private void loadStateIntoEventCreator(@NonNull final SignedState signedState) {
         Objects.requireNonNull(signedState);
 
-        if (tipsetEventCreator == null) {
+        if (eventCreator == null) {
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java:774]: This check isn't necessary anymore

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java [hunk 2]
```diff
@@ -1003,7 +1003,7 @@ public void start() {
         // The event creator is intentionally started before replaying the preconsensus event stream.
         // This prevents the event creator's intake queue from filling up and blocking. Note that
         // this component won't actually create events until the platform has the appropriate status.
-        Optional.of(tipsetEventCreator).ifPresent(Startable::start);
+        Optional.of(eventCreator).ifPresent(Startable::start);
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java:1006]: Optional no longer required

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java [hunk 3]
```diff
@@ -1020,7 +1020,7 @@ public void start() {
      */
     public void performPcesRecovery() {
         components.start();
-        Optional.of(tipsetEventCreator).ifPresent(Startable::start);
+        Optional.of(eventCreator).ifPresent(Startable::start);
```
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java:1023]: Optional no longer required

**Review by alittley — DISMISSED** [review]

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java (hunk 1 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java:774]: fixed

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java (hunk 2 above)
Comment (cody-littley) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java:1006]: fixed

**Review by alittley — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by alittley — APPROVED** [review]

**Pull request merged**